pycds/weather_anomaly.py

```diff
@@ -63,13 +75,29 @@
 # TODO: Factor out common query structure into a defined function (parameterized by min/max function [can this be done?]
 # and by cell_method
 
-class DailyMaxTemperature(Base, MaterializedViewMixin):
-    __selectable__ = text('''
+def daily_temperature_extremum_selectable(extremum):
+    '''Return a SQLAlchemy selector for a specified extremum of daily temperature.
+
+    Args:
+        extremum (str): 'max' | 'min'
+
+    Returns:
+        sqlalchemy.sql.expression.FromClause (in fact, sqlalchemy.sql.expression.TextClause)
+    '''
+
+    def cell_method(extremum):
+        method = {
+            'max': ['time: maximum', 'time: point', 'time: mean'],
+            'min': ['time: minimum', 'time: point', 'time: mean'],
+        }
+        return ', '.join("'{}'".format(m) for m in method[extremum])
```

Comment:
Couldn't this whole thing just be:
```
"'time: {}imum', 'time:point', 'time:mean'".format(extremum)
```
I think that you could in inline it below...